index.js

```diff
@@ -111,7 +129,19 @@ module.exports = function (config) {
       $defs.remove()
     }
 
-    $symbol.append($svg.contents())
+    let $groupWrap = null
+    for (let [name, value] of Object.entries($svg.attr())) {
```

Comment:
It's a bummer that cheerio doesn't provide an API to get an array of attributes. What we could do instead, is looping through `presentationAttributes` (if we keep it as array) and collect attributes with values. Would something like this work?

```
let $groupWrap = null;
for (let attrName of presentationAttributes) {
   const attrValue = $svg.attr(name);
   if (attrValue === undefined) continue;
   if (!$groupWrap) $groupWrap = $('<g/>');
   $groupWrap.attr(attrName, attrValue);
}
```

Reply:
And what's wrong with `Object.entries($svg.attr())`?

If it's speed, with `$svg.attr()` we are looping through a couple attributes for each `$svg`. With `for...of presentationAttributes`, we're going to loop through 65 attributes each time.

Reply:
It's more about `presentationAttributes.indexOf(name) === -1` in your implementation that has to loop through all presentation attributes each time.

Reply:
Oh right, brainfart on my part.

Set might still be faster though. And when looping through `presentationAttributes`, the order of the attributes on svg is not transfered to the `<g>`. So for example it's harder to write tests for it.

Reply:
I see. Hmm, then maybe go with my original proposal to use `Set`?